### DISCUSSION TITLE
SmartCard-Passthrough.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#6171] [Ideas]
As we are in progress changing everything to YubiKey-Logon, we also use the SmartCard-Logon-Feature for our Windows Servers.
Would be great if we can pass through the Smartcard from Browser to the Server in MeshCentral which we remote control. 

But no idea if that is possible at all.

**Comment by silversword411** [comment]
Put authentic in front of mesh and pipe yubikey thru that
